python/taichi/misc/gui.py

```diff
@@ -19,13 +20,16 @@ class Event:
     LEFT = 'Left'
     RIGHT = 'Right'
     CAPSLOCK = 'Caps_Lock'
-    MOTION = 'Motion'
     LMB = 'LMB'
     MMB = 'MMB'
     RMB = 'RMB'
     EXIT = 'WMClose'
-    RELEASE = False
-    PRESS = True
+    WHEEL = 'WHEEL'
+
+    # Event types
+    MOTION = None  # KeyEvent.EType.Move
+    PRESS = None  # KeyEvent.EType.Press
+    RELEASE = None  # KeyEvent.EType.Release
```

Comment:
```suggestion
    MOTION = ti.ti_core.KeyEvent.EType.Move
    PRESS = ti.ti_core.KeyEvent.EType.Press
    RELEASE = ti.ti_core.KeyEvent.EType.Release
```
IIRC `ti.ti_core` is loaded before `misc.gui` in `python/taichi/__init__.py`.